hawkular-inventory-parent/hawkular-inventory-api/src/main/java/org/hawkular/inventory/api/InventoryService.java

```diff
@@ -72,20 +72,22 @@
     /**
      * Get a list of resources with filters and default pagination options
      * @param root true to get only root resources
+     * @param feedId filter with that feed ID, or null for no type filter
      * @param typeId filter with that type ID, or null for no type filter
      * @return resources embedded in page object {@link ResultSet}
      */
-    ResultSet<ResourceWithType> getResources(boolean root, String typeId);
+    ResultSet<ResourceWithType> getResources(boolean root, String feedId, String typeId);
```

Comment:
I thought the feedId was necessary in the model (to build a bridge with CMDGW), but not for queries

Reply:
No, it is, the backfill feature in hServices has a explicit query "give me all resources under that feed".
Just trying to code it without major refactor is why I introduced this now.